.github/CODEOWNERS

```diff
@@ -989,6 +989,13 @@
 # ServiceLabel: %Spring App Discovery %Mgmt
 # ServiceOwners:                                                   @sunkun99
 
+# PRLabel: %Informaticadatamanagement
```

Comment:
Please move this section up to ~L950 to preserve alphabetical ordering by label name in the management section.